airbyte-integrations/connectors/source-chargebee/source_chargebee/schemas/customer.json

```diff
@@ -175,6 +175,15 @@
     "object": {
       "type": ["string", "null"]
     },
+    "mrr": {
+      "type": ["integer", "null"]
+    },
+    "tax_providers_fields": {
+      "type": ["array", "null"],
+      "items": {
+        "type": ["integer", "string", "null"]
```

Comment:
Type cannot be declared in that way

Reply:
@artem1205 Is this due to the way Airbyte handles converting data types?

Reply:
Yep, this may cause an issue in destination connectors

Reply:
Removed in latest commit